plugins/dex/list/handler.go

```diff
@@ -22,8 +22,14 @@ func NewHandler(keeper *order.DexKeeper, tokenMapper tokens.Mapper, govKeeper go
 	return func(ctx sdk.Context, msg sdk.Msg) sdk.Result {
```

Comment:
Related questions：
1）Should we also disable cli `directly` (e.g., ListMiniMsg) ?
2) Should we also revise the logic for handling proposal?

Reply:
sorry, maybe disable cli is not easy.

Reply:
1. The server will return an error, no need to modify the cli.
2. The associated proposal will be useless after we disable placing new orders. No need to revise the proposal handler.